### ISSUE TITLE
xbrowsersync claims "Invalid credentials"

### ISSUE THREAD

**Issue by Kellegram** [xbrowsersync/api#66]
I have been using xbrowsersync via browser extension for many months now, using the main server for sync. I have the credentials in the password manager and used them many times. I suddenly cannot login with those credentials anymore, been like this for a few weeks now, I was hoping it was a temporary issue, but it's not. I know for a fact the credentials I am using are correct, but it claims they are not.

**Comment by Avinash-Lvs** [comment]
have you stopped using the extension for 21days. If thats the case, it is evident that your account has been deleted automatically.

**Comment by Kellegram** [comment]
> have you stopped using the extension for 21days. If thats the case, it is evident that your account has been deleted automatically.

No, it's why I said suddenly. It happened out of nowhere. I wake up, open up my laptop and see that I am not synced. I try to login (100% correct credentials as I use a login manager) and nothing. I haven't used it since this post since I can't really risk losing my bookmarks, I was lucky I had a backup so I only lost some % of the newer ones.

**Comment by aSapien** [comment]
Same issue here. My old computer died and I was hoping that XBS would save the day. I had lost years of carefully curated bookmarks.

**Comment by stefanosandes** [comment]
Same problem here!

**Comment by retrogamer0815** [comment]
Same problem here, just wanted to set it up in my browser on a new computer and I get the "invalid credentials" message.
Since someone above asked the question about having stopped usage of the extension for 21 days, this is definitely not the case. It might be, that I didn't add or delete any bookmarks for three weeks, but using the browser with the extension should still count as usage.

But still I feel that deleting accounts after just 21 days sounds a little fast. I use the extension to sync the bookmarks between my laptop and a desktop computer and no other devices. Otherwise I use an iPhone and an iPad where xbrowsersync is not available, however this is no problem for me, since the use cases for my computers and mobile devices are completely different.
However I can see myself in the future going on a vacation for a few weeks and just taking my phone and iPad with me, having my account deleted in such a case just for 21 days not using the extension would be a little harsh I feel.

**Comment by cybergrrl** [comment]
this happens to me every few months. forces me to make a new id and password. i too use a password manager, so it is not about typos. has it ever been resolved?

**Comment by keizah7** [comment]
same issue. even using API itself. GET /bookmarks/{id}
